VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/select-hearing-layout.component.spec.ts

```diff
@@ -36,6 +37,17 @@ describe('SelectHearingLayoutComponent', () => {
         expect(component.selectedLayout).toBe(layout);
     });
 
+    it('should call translate service to update hearing layout option when language changes', () => {
+        component.currentText = textButton.innerHTML.toLowerCase().split(' ').join('-');
+        component.accordionOpenAllElement = document.getElementsByClassName('govuk-accordion__open-all').item(0) as HTMLButtonElement;
+        component.ngOnInit();
+        onLangChangeSpy.emit({ lang: 'tl' } as LangChangeEvent);
+        expect(component.currentText).toBe('open-all');
+        expect(translateServiceSpy.instant).toHaveBeenCalledWith('select-hearing-layout.choose-hearing-layout');
+        expect(translateServiceSpy.instant).toHaveBeenCalledWith(`select-hearing-layout.${component.currentText}`);
```

Comment:
I would make translateServiceSpy.instant return a specific value we can test for when called by this string.

Reply:
ok cool, I will attempt to do that.

Reply:
https://stackoverflow.com/questions/16198353/any-way-to-modify-jasmine-spies-based-on-arguments